servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/DefaultReadOnlyTcpServerConfig.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.tcp.netty.internal;
+
+import io.servicetalk.transport.api.ServerSslConfig;
+import io.servicetalk.transport.api.SslListenMode;
+import io.servicetalk.transport.api.TransportObserver;
+import io.servicetalk.transport.netty.internal.NoopTransportObserver;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.ssl.SslContext;
+import io.netty.util.DomainWildcardMappingBuilder;
+import io.netty.util.Mapping;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Map.Entry;
+import javax.annotation.Nullable;
+
+import static io.servicetalk.transport.api.TransportObservers.asSafeObserver;
+import static io.servicetalk.transport.netty.internal.SslContextFactory.forServer;
+
+/**
+ * Read only view of {@link TcpServerConfig}.
+ */
+public final class DefaultReadOnlyTcpServerConfig extends AbstractReadOnlyTcpConfig<ServerSslConfig>
```

Comment:
Identical to the old class of ReadOnlyTcpServerConfig, just added a Default in the name